lms/djangoapps/courseware/views/index.py

```diff
@@ -388,9 +388,15 @@ def _create_courseware_context(self):
             if self.section.position and self.section.has_children:
                 display_items = self.section.get_display_items()
                 if display_items:
-                    courseware_context['sequence_title'] = display_items[self.section.position - 1] \
-                        .display_name_with_default
-
+                    try:
+                        courseware_context['sequence_title'] = display_items[self.section.position - 1] \
+                            .display_name_with_default
+                    except IndexError:
+                        log.info("Course section {} with position {} and total section display items: {}".format(
```

Comment:
@Rabia23 , you'll want to raise the error after you catch and log it. Right now, this view wouldn't raise an error. It should. I wonder, what happens now in this view if we hit this bug?

@Ayub-Khan , FYI

The pattern you want is:

```python
try:
  do_thing()
except IndexError:
  log.info(...)
  raise
```